Source/SwiftyDropbox/Platform/SwiftyDropbox_iOS/OAuthMobile.swift

```diff
@@ -292,7 +292,7 @@ open class MobileSharedApplication: SharedApplication {
     public static var sharedMobileApplication: MobileSharedApplication?
 
     let sharedApplication: UIApplication
-    let controller: UIViewController?
+    weak var controller: UIViewController?
```

Comment:
the weak controller _might_ cause problems... I remember seeing it and wanting to change it, but not sure why I left it alone